hapi-server/src/main/java/com/example/fhirexample/ResourceProvider/PractitionerResourceProvider.java

```diff
@@ -184,85 +89,44 @@ public Practitioner read(@IdParam IdType theId) {
     }
 
     @Search
-    public List<DomainResource> findPractitionersByGiven(
-            @RequiredParam(name=Practitioner.SP_GIVEN) StringAndListParam paramValues,
-            @Offset Integer theOffset,
-            @Count Integer theCount,
-            @IncludeParam(reverse=true, allow={"PractitionerRole:practitioner"}) Set<Include> theReverseIncludes) {
-        List<DomainResource> results = new ArrayList<>();
-        Pagination page = new Pagination(theOffset, theCount);
-        try {
-            ObjectMapper objectMapper = new ObjectMapper();
-            List<SearchCriteria> searchCriteriaList = searchCriteria(Practitioner.SP_GIVEN, paramValues);
+    public List<DomainResource> search(
+        @OptionalParam(name = Practitioner.SP_RES_ID) TokenAndListParam id,
+        @OptionalParam(name = Practitioner.SP_IDENTIFIER) TokenAndListParam identifier,
+        @OptionalParam(name = Practitioner.SP_FAMILY) StringAndListParam family,
+        @OptionalParam(name = Practitioner.SP_NAME) StringAndListParam name,
+        @OptionalParam(name = Practitioner.SP_GIVEN) StringAndListParam given,
+        @OptionalParam(name = Constants.PARAM_LASTUPDATED) DateRangeParam lastUpdated,
```

Comment:
Instead of using a DateRange param, use the DateParam. This will allow {<, <=, =, >=, >} options for the modifiers and will send those modifiers to the DS. we could also update this to be DateAndListParam to allow more options for the query (i.e. more than 1 criteria for the date parameter allowing date ranges in addition to the other options)